sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -339,10 +339,15 @@ class HiveSparkSubmitSuite
   private def runSparkSubmit(args: Seq[String]): Unit = {
     val sparkHome = sys.props.getOrElse("spark.test.home", fail("spark.test.home is not set!"))
     val history = ArrayBuffer.empty[String]
-    val commands = Seq("./bin/spark-submit") ++ args
+    val sparkSubmit = if (Utils.isWindows) {
```

Comment:
I think we don't have such ones anymore assuming from the rest of errors - [here](https://gist.github.com/HyukjinKwon/88a0b37cd027934bc14f3aa9f812be31).

We have a similar ones that are failed during trying to execute `/bin/bash` but I believe they are different with this case.